### ISSUE TITLE
DOLFIN has been configured without support for SCOTCH

### ISSUE THREAD

**Issue by tengzhangjj** [Juanlu001/fenics-recipes#7]
Fenics 1.4 installed with conda can only run in serial now, there will be error of missing SCOTCH when running in parallel:

"Reason:  DOLFIN has been configured without support for SCOTCH"

Thanks a lot.

**Comment by astrojuanlu** [comment]
I spent some time trying to get SCOTCH 6.0 to work and I am still failing. I am not sure I will be able to deliver the packages this weekend, perhaps I will need some more time.

**Comment by tengzhangjj** [comment]
Thank you very much for your efforts on this.

Best regards,
Teng

2015-04-10 17:37 GMT-04:00 Juan Luis Cano Rodríguez <
notifications@github.com>:

**Comment by astrojuanlu** [comment]
Closing this in favor of #29.
